.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
It's a GitHub standard, just not enforced by Drupal standards so much I believe, but since the dev work is GH based, we will follow. Just go to end of line and press Enter. It's just a return char at the end of last line.

E.g. reason: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file